uploadNeuroDB/tarchive_validation.pl

```diff
@@ -304,20 +310,6 @@ =head2 Methods
     $subjectIDsref, $sex, \%tarchiveInfo, $User, $centerID, $upload_id
 );
 
-################################################################
-################################################################
-## Check the CandID/PSCID Match It's possible that the CandID ## 
-## exists, but doesn't match the PSCID. This will fail further #
-## down silently, so we explicitly check that the data is ######
-## correct here. ###############################################
-################################################################
-################################################################
-my $CandMismatchError= $utility->validateCandidate($subjectIDsref);
```

Comment:
removed the call to `validateCandidate()` as it is now part of the determineSubjectIDs function and is returned into $subjectIDsref->{'CandMismatchError'}